superset-frontend/plugins/preset-chart-xy/src/components/Line/Encoder.ts

```diff
@@ -46,7 +46,7 @@ export const lineEncoderFactory = createEncoderFactory<LineEncodingConfig>({
     x: { field: 'x', type: 'quantitative' },
     y: { field: 'y', type: 'quantitative' },
     fill: { value: false, legend: false },
-    stroke: { value: '#222' },
+    stroke: { value: '#222' }, // eslint-disable-line theme-colors/no-literal-colors
```

Comment:
I disabled the rule here because it seems this `Line` chart is deprecated. Can you confirm @villebro @zhaoyongjie?